docs/migration/migrate_7_0.asciidoc

```diff
@@ -148,6 +149,23 @@ generalized from the source data for visualization and locations are approximate
 Codes layer will appear differently. ZCTA boundaries have changed dramatically 
 in some areas. Also, some zip codes have been deprecated and new ones have been added.
 
+
+[float]
+[[breaking_70_index_pattern_changes]]
+=== Index pattern changes 
+
+[float]
+==== Time-based internal index patterns queried as wildcard index patterns
+*Details:* Time-based interval index patterns were deprecated in 5.x. In 6.x, 
+you could no longer create time-based interval index patterns, but they continued 
+to function as expected. In 7.0, functionality has changed such that {kib} will 
+automatically query a time-based interval index pattern as a wildcard index pattern.
+
+*Impact:* If you query a time-based index pattern, that query will now be performed 
+on the root wildcard term. For example, a query on `logstash-2019-04-18` will now 
```

Comment:
Can we change this to something like "For example, a query on an index pattern such as `[logstash-]YYYY.MM.DD` will now query all indices that match `logstash-*`."

Reply:
@lukasolson Sure, I made that change. I also added a note about preparing for full support being removed in 8.0. Do you think we should include that? If so, can you confirm where users can migrate these indices? I think you mentioned it was under Advanced Settings, but I don't see it there. Thanks.

Reply:
@lukasolson Is this breaking change still going in for 7.0.1? If so, we should merge this before Wednesday. Do you have time to review today?